primitives/consensus/common/src/import_queue/basic_queue.rs

```diff
@@ -46,7 +48,8 @@ pub struct BasicQueue<B: BlockT, Transaction> {
 impl<B: BlockT, Transaction> Drop for BasicQueue<B, Transaction> {
 	fn drop(&mut self) {
 		// Flush the queue and close the receiver to terminate the future.
-		self.sender.close_channel();
+		self.finality_sender.close_channel();
```

Comment:
https://docs.rs/futures-channel/0.3.6/src/futures_channel/mpsc/mod.rs.html#991

Actually that is done automatically :P

Reply:
I guess this was done explicitly in case there's more than one instance of the sender. I don't really have a strong opinion on this.